cassandra-scheduler/src/test/java/com/mesosphere/dcos/cassandra/common/offer/ClusterTaskOfferRequirementProviderTest.java

```diff
@@ -234,16 +234,19 @@ public void testGetNewOfferRequirement() throws Exception {
         final Protos.ExecutorInfo executorInfo = requirement.getExecutorRequirementOptional().get().getExecutorInfo();
```

Comment:
The `executorInfo` now has one more URI (the location of the libmesos bundle) so the tests needed to reflect that.